Update PUID to 80X training

jdolen opened the pull request:
New PUID 80X training has been added to cms-data:
https://github.com/cms-data/RecoJets-JetProducers/pull/8
This PR updates the CMSSW configs.

A summary of the 80X training results can be found here:
﻿﻿https://www.dropbox.com/s/x3wvesbyavtiqg3/PUIDpres1.pdf?dl=0

Further validation is in progress.

jdolen committed:
adding 80x puid configs

cmsbuild commented:
A new Pull Request was created by  @jdolen (James Dolen) for CMSSW_8_0_X.

It involves the following packages:

RecoJets/JetProducers


@cmsbuild, @cvuosalo, @slava77, @davidlange6 can you please review it and eventually sign? Thanks.
@TaiSakuma, @ahinzmann, @rappoccio, @yslai, @nhanvtran, @schoef, @mariadalfonso this is something you requested to watch as well.
@slava77, @degano, @smuzaffar you are the release manager for this.

cms-bot commands are list here cms-sw/cmssw#13028

slava77 commented on RecoJets/JetProducers/python/PileupJetID_cfi.py:
```diff
@@ -9,6 +9,7 @@
 _chsalgos_76x = cms.VPSet(full_76x_chs,cutbased)
+_chsalgos_80x = cms.VPSet(full_80x_chs,cutbased)
 
 _stdalgos    = _chsalgos_76x
```
I suppose, this means that the default behavior is unchanged.
This update just introduces selections for the new training/working point, which will need to be enabled when?

ahinzmann replied:
This is a bug, it should indeed be enabled now.

slava77 commented:
@jdolen we also need a PR in 81X.

jdolen committed:
Fixing to run 80x algo

cmsbuild commented:
Pull request #14043 was updated. @cmsbuild, @cvuosalo, @slava77, @davidlange6 can you please check and sign again.

slava77 commented:
@cmsbuild please test

please provide a PR for 81X.
It should be reviewed and integrated first.

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
-1 
Tested at: f0a06593b8f4f4653623aea63c007df2d6bfdb16
 When I ran the RelVals I found an error in the following worklfows: 
 1003.0 step2
<pre>runTheMatrix-results/1003.0_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM/step2_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM.log

----- Begin Fatal Exception 13-Apr-2016 16:12:13 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1003.0_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM

----- End Fatal Exception -------------------------------------------------

</pre>
1001.0 step2
<pre>runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4/step2_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4.log

----- Begin Fatal Exception 13-Apr-2016 16:12:16 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4

----- End Fatal Exception -------------------------------------------------

</pre>
1000.0 step2
<pre>runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT.log

----- Begin Fatal Exception 13-Apr-2016 16:12:18 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT

----- End Fatal Exception -------------------------------------------------

</pre>
1306.0 step3
<pre>runTheMatrix-results/1306.0_SingleMuPt1_UP15+SingleMuPt1_UP15+DIGIUP15+RECOUP15+HARVESTUP15/step3_SingleMuPt1_UP15+SingleMuPt1_UP15+DIGIUP15+RECOUP15+HARVESTUP15.log

----- Begin Fatal Exception 13-Apr-2016 16:17:10 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1306.0_SingleMuPt1_UP15+SingleMuPt1_UP15+DIGIUP15+RECOUP15+HARVESTUP15

----- End Fatal Exception -------------------------------------------------

</pre>
134.911 step3
<pre>runTheMatrix-results/134.911_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step3_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT.log

----- Begin Fatal Exception 13-Apr-2016 16:17:34 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/134.911_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT

----- End Fatal Exception -------------------------------------------------

</pre>
9.0 step3
<pre>runTheMatrix-results/9.0_Higgs200ChargedTaus+Higgs200ChargedTaus+DIGI+RECO+HARVEST/step3_Higgs200ChargedTaus+Higgs200ChargedTaus+DIGI+RECO+HARVEST.log

----- Begin Fatal Exception 13-Apr-2016 16:18:58 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/9.0_Higgs200ChargedTaus+Higgs200ChargedTaus+DIGI+RECO+HARVEST

----- End Fatal Exception -------------------------------------------------

</pre>
1330.0 step3
<pre>runTheMatrix-results/1330.0_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15/step3_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15.log

----- Begin Fatal Exception 13-Apr-2016 16:19:28 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1330.0_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15

----- End Fatal Exception -------------------------------------------------

</pre>
135.4 step3
<pre>runTheMatrix-results/135.4_ZEE_13+ZEEFS_13+HARVESTUP15FS+MINIAODMCUP15FS/step3_ZEE_13+ZEEFS_13+HARVESTUP15FS+MINIAODMCUP15FS.log

----- Begin Fatal Exception 13-Apr-2016 16:20:16 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetId'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/135.4_ZEE_13+ZEEFS_13+HARVESTUP15FS+MINIAODMCUP15FS

----- End Fatal Exception -------------------------------------------------

</pre>
25.0 step3
<pre>runTheMatrix-results/25.0_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT/step3_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT.log

----- Begin Fatal Exception 13-Apr-2016 16:20:36 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/25.0_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT

----- End Fatal Exception -------------------------------------------------

</pre>
50202.0 step3
<pre>runTheMatrix-results/50202.0_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50/step3_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50.log

----- Begin Fatal Exception 13-Apr-2016 16:27:18 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/50202.0_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50

----- End Fatal Exception -------------------------------------------------

</pre>
25202.0 step3
<pre>runTheMatrix-results/25202.0_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25/step3_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25.log

----- Begin Fatal Exception 13-Apr-2016 16:30:54 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw-patch/CMSSW_8_0_X_2016-04-12-1100/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/25202.0_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25

----- End Fatal Exception -------------------------------------------------

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14043/12346/summary.html 

The following merge commits were also included on top of IB + this PR after doing git cms-merge-topic: 
https://github.com/cms-sw/cmssw/commit/b9a53a361aafe40f726a473d2fddb8cdeccd3120
https://github.com/cms-sw/cmssw/commit/424bec27905e3d9afa29c68fe7b0e043b9144303
https://github.com/cms-sw/cmssw/commit/640507c16aabf063c277b59e14bfa7808eb12af5
https://github.com/cms-sw/cmssw/commit/17ab3e96ef8139dc14098336f20c7f74f4ca5a39
You can see more details here:
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14043/12346/git-log-recent-commits
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14043/12346/git-merge-result

  'List

slava77 commented:
Looks like  the new IB in 80X is not out yet. :clock11:

slava77 commented:
@cmsbuild please test
8_0_X_2016-04-12-2300 is now available

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
-1 
Tested at: f0a06593b8f4f4653623aea63c007df2d6bfdb16
 When I ran the RelVals I found an error in the following worklfows: 
 1001.0 step2
<pre>runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4/step2_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4.log

----- Begin Fatal Exception 14-Apr-2016 00:57:31 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1001.0_RunMinBias2011A+RunMinBias2011A+TIER0EXP+ALCAEXP+ALCAHARVD1+ALCAHARVD2+ALCAHARVD3+ALCAHARVD4

----- End Fatal Exception -------------------------------------------------

</pre>
1003.0 step2
<pre>runTheMatrix-results/1003.0_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM/step2_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM.log

----- Begin Fatal Exception 14-Apr-2016 00:57:33 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1003.0_RunMinBias2012A+RunMinBias2012A+RECODDQM+HARVESTDDQM

----- End Fatal Exception -------------------------------------------------

</pre>
1000.0 step2
<pre>runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT/step2_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT.log

----- Begin Fatal Exception 14-Apr-2016 00:57:34 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1000.0_RunMinBias2011A+RunMinBias2011A+TIER0+SKIMD+HARVESTDfst2+ALCASPLIT

----- End Fatal Exception -------------------------------------------------

</pre>
1306.0 step3
<pre>runTheMatrix-results/1306.0_SingleMuPt1_UP15+SingleMuPt1_UP15+DIGIUP15+RECOUP15+HARVESTUP15/step3_SingleMuPt1_UP15+SingleMuPt1_UP15+DIGIUP15+RECOUP15+HARVESTUP15.log

----- Begin Fatal Exception 14-Apr-2016 00:59:39 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1306.0_SingleMuPt1_UP15+SingleMuPt1_UP15+DIGIUP15+RECOUP15+HARVESTUP15

----- End Fatal Exception -------------------------------------------------

</pre>
134.911 step3
<pre>runTheMatrix-results/134.911_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT/step3_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT.log

----- Begin Fatal Exception 14-Apr-2016 01:00:29 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/134.911_RunSinglePh2015D+RunSinglePh2015D+HLTDR2_25ns+RECODR2_25nsreHLT+HARVESTDR2_25nsreHLT

----- End Fatal Exception -------------------------------------------------

</pre>
1330.0 step3
<pre>runTheMatrix-results/1330.0_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15/step3_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15.log

----- Begin Fatal Exception 14-Apr-2016 01:02:13 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/1330.0_ZMM_13+ZMM_13+DIGIUP15+RECOUP15+HARVESTUP15

----- End Fatal Exception -------------------------------------------------

</pre>
135.4 step3
<pre>runTheMatrix-results/135.4_ZEE_13+ZEEFS_13+HARVESTUP15FS+MINIAODMCUP15FS/step3_ZEE_13+ZEEFS_13+HARVESTUP15FS+MINIAODMCUP15FS.log

----- Begin Fatal Exception 14-Apr-2016 01:02:27 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetId'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/135.4_ZEE_13+ZEEFS_13+HARVESTUP15FS+MINIAODMCUP15FS

----- End Fatal Exception -------------------------------------------------

</pre>
9.0 step3
<pre>runTheMatrix-results/9.0_Higgs200ChargedTaus+Higgs200ChargedTaus+DIGI+RECO+HARVEST/step3_Higgs200ChargedTaus+Higgs200ChargedTaus+DIGI+RECO+HARVEST.log

----- Begin Fatal Exception 14-Apr-2016 01:02:36 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/9.0_Higgs200ChargedTaus+Higgs200ChargedTaus+DIGI+RECO+HARVEST

----- End Fatal Exception -------------------------------------------------

</pre>
25.0 step3
<pre>runTheMatrix-results/25.0_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT/step3_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT.log

----- Begin Fatal Exception 14-Apr-2016 01:03:33 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/25.0_TTbar+TTbar+DIGI+RECOAlCaCalo+HARVEST+ALCATT

----- End Fatal Exception -------------------------------------------------

</pre>
50202.0 step3
<pre>runTheMatrix-results/50202.0_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50/step3_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50.log

----- Begin Fatal Exception 14-Apr-2016 01:10:25 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/50202.0_TTbar_13+TTbar_13+DIGIUP15_PU50+RECOUP15_PU50+HARVESTUP15_PU50

----- End Fatal Exception -------------------------------------------------

</pre>
25202.0 step3
<pre>runTheMatrix-results/25202.0_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25/step3_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25.log

----- Begin Fatal Exception 14-Apr-2016 01:11:36 CEST-----------------------

An exception of category 'FileInPathError' occurred while

   [0] Constructing the EventProcessor

   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluatorCHSDQM'

Exception Message:

edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80x_Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.

The search path is defined by: CMSSW_SEARCH_PATH

${CMSSW_SEARCH_PATH} is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/src:/build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/src:/cvmfs/cms-ib.cern.ch/week0/slc6_amd64_gcc530/cms/cmssw/CMSSW_8_0_X_2016-04-12-2300/external/slc6_amd64_gcc530/data

Current directory is: /build/cmsbuild/jenkins-workarea/workspace/ib-any-integration/runTheMatrix-results/25202.0_TTbar_13+TTbar_13+DIGIUP15_PU25+RECOUP15_PU25+HARVESTUP15_PU25

----- End Fatal Exception -------------------------------------------------

</pre>

 you can see the results of the tests here: 
 https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14043/12355/summary.html 
  'List

slava77 commented:
@degano
Was the cmsdist for the data package merged? I thought it went in at the
same time as 81x.

davidlange6 commented:
In the data of the data files, 80x should be 80X to match what is in the data package
(this is different from the naming convention used in the 76x files…)

> On Apr 14, 2016, at 3:47 AM, Slava Krutelyov <notifications@github.com> wrote:
> 
> @degano
> Was the cmsdist for the data package merged? I thought it went in at the
> same time as 81x.
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
>

jdolen committed:
Fixing filenames to match cms-data

cmsbuild commented:
Pull request #14043 was updated. @cmsbuild, @cvuosalo, @slava77, @davidlange6 can you please check and sign again.

slava77 commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

cmsbuild commented:
+1
Tested at: 0e368c334b5be81d227116522a82e6eeeb58c16a
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14043/12382/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14043/12382/summary.html

slava77 commented:
Looking at jenkins tests comparisons,  there are changes in the PU ID discriminant distributions (expected variables changed).


The direction of the change appears to be opposite to what's in the PUIDpres1.pdf linked in this PR description:
<img width="837" alt="trainingp4grab" src="https://cloud.githubusercontent.com/assets/4676718/14549777/f1d57110-0277-11e6-85d0-bffe946cdf90.png">
this tells me that there should generally be less events with new training above a given value of MVA (solid  curves of the same color are always below the dashed ones) for all inputs.

however , e.g. in 134.911_RunSinglePh2015D workflow, the MVA plot shows that there are more events above a given value of MVA with the new training
<img width="880" alt="wf134 911_puid_lopt_fwd_mva" src="https://cloud.githubusercontent.com/assets/4676718/14549861/8dadc920-0278-11e6-9340-c2a53a98907b.png">

@jdolen @ahinzmann 
please clarify the relationship
(I'm likely misinterpreting these rather different plots; and curves in PUIDpres1.pdf  do not have a legend)

jdolen commented:
In the forward region I would expect more jets with low discriminant with the 8X training, but I imagine this also depends on the sample composition. Could you confirm that in the DQM plot red is 80X and black is 76X?

Here are the discriminant for a DY sample:
![puidpres2 001](https://cloud.githubusercontent.com/assets/4482920/14553204/3aa65652-02aa-11e6-9d0d-fd16d848101e.jpeg)
![puidpres2 002](https://cloud.githubusercontent.com/assets/4482920/14553205/3f022460-02aa-11e6-9733-88d05f3cc445.jpeg)

slava77 commented:
On 4/14/16 11:38 PM, James Dolen wrote:
> In the forward region I would expect more jets with low discriminant
> with the 8X training, but I imagine this also depends on the sample
> composition. Could you confirm that in the DQM plot red is 80X and black
> is 76X?

Yes, red is for this PR and black is for the baseline release version

Are your plots made directly with the code in this PR + the training
files from the release,
or was it a private version?
If you can suggest a sample that would be more clear to show the effect
in the standard DQM plots, I can run on it to check.

jdolen committed:
Fixing PUID working points 

Calculated using the correct input file (80X trained PUMVA)

cmsbuild commented:
Pull request #14043 was updated. @cmsbuild, @cvuosalo, @slava77, @davidlange6 can you please check and sign again.

slava77 commented:
@cmsbuild please test

cmsbuild commented:
The tests are being triggered in jenkins.

jdolen commented:
Hi @slava77 . Thank you for pointing out that the discriminant change in this PR did not match my validation plots. There was an error in my plots. The training is correct, but I accidentally included plots from 80X samples running with the 76X training instead of the 80X training. I've updated the plots below, and also updated the working points to those derived from the 80X training.

![puidpres3_80xib 001](https://cloud.githubusercontent.com/assets/4482920/14579603/483f8dc6-0375-11e6-8007-ab5106d290ea.jpeg)
![puidpres3_80xib 002](https://cloud.githubusercontent.com/assets/4482920/14579605/4c3ba8e2-0375-11e6-8322-9df417719fa5.jpeg)
![puidpres3_80xib 003](https://cloud.githubusercontent.com/assets/4482920/14579607/51d45588-0375-11e6-8e0c-c15e31e28015.jpeg)
![puidpres3_80xib 004](https://cloud.githubusercontent.com/assets/4482920/14579608/54493c16-0375-11e6-9cb8-0cadbfc32d1c.jpeg)
![puidpres3_80xib 005](https://cloud.githubusercontent.com/assets/4482920/14579609/56cc3f60-0375-11e6-9ade-d2fc641abd01.jpeg)
![puidpres3_80xib 006](https://cloud.githubusercontent.com/assets/4482920/14579611/591c30d6-0375-11e6-8df9-9caf3ec0acb6.jpeg)
![puidpres3_80xib 007](https://cloud.githubusercontent.com/assets/4482920/14579614/5b958628-0375-11e6-986f-74ccd274e5d1.jpeg)
![puidpres3_80xib 008](https://cloud.githubusercontent.com/assets/4482920/14579615/5d9c8f98-0375-11e6-8850-b3475c29c54d.jpeg)

cmsbuild commented:
+1
Tested at: a8e72899034c98ca4efcb4fdddb914602f640ca9
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14043/12428/summary.html

cmsbuild commented:
Comparison is ready
https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-14043/12428/summary.html

slava77 commented:
+1

for https://github.com/cms-sw/cmssw/pull/14043	a8e7289
- updated training inputs and working point selections for 80X training; in line with the description of the PR
- jenkins tests pass and comparisons with baseline show differences in PU JetID variables in line with the latest internal validation plots https://github.com/cms-sw/cmssw/pull/14043#issuecomment-210748704

cmsbuild commented:
This pull request is fully signed and it will be integrated in one of the next CMSSW_8_0_X IBs (tests are also fine). This pull request requires discussion in the ORP meeting before it's merged. @slava77, @davidlange6, @degano, @smuzaffar

Martin-Grunewald commented:
Looks like this PR creates the large-scale relval failure:
```
----- Begin Fatal Exception 19-Apr-2016 10:04:18 CEST-----------------------
An exception of category 'FileInPathError' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing module: class=PileupJetIdProducer label='pileupJetIdEvaluat\
orCHSDQM'
Exception Message:
edm::FileInPath unable to find file RecoJets/JetProducers/data/pileupJetId_80X_\
Eta0to2p5_BDT.weights.xml.gz anywhere in the search path.
The search path is defined by: CMSSW_SEARCH_PATH
${CMSSW_SEARCH_PATH} is: /data/CMS0/CMSSW_8_0_X_2016-04-18-2300/src:/data/CMS0/\
CMSSW_8_0_X_2016-04-18-2300/external/slc6_amd64_gcc493/data:/cvmfs/cms-ib.cern.\
ch/week1/slc6_amd64_gcc493/cms/cmssw-patch/CMSSW_8_0_X_2016-04-18-2300/src:/cvm\
fs/cms-ib.cern.ch/week1/slc6_amd64_gcc493/cms/cmssw-patch/CMSSW_8_0_X_2016-04-1\
8-2300/external/slc6_amd64_gcc493/data
Current directory is: /data/CMS0/CMSSW_8_0_X_2016-04-18-2300/src/HLTrigger/Conf\
iguration/test
----- End Fatal Exception -------------------------------------------------
```
??

slava77 commented:
@Martin-Grunewald 
which workflow/ARCH is this for?
A lot of workflows  fail in  CMSSW_8_0_X_2016-04-18-1100  in the following
```
   [6] Calling produce method for unscheduled module ShiftedPATJetProducer/'shiftedPatJetEnUpPuppi'
Exception Message:
 cannot find key 9 in the JEC payload, this usually means you have to change the global tag
```

...
I found the error you mentioned only in slc6_amd64_gcc493 in CMSSW_8_0_X_2016-04-18-2300.
This looks like cvmfs sync problem

Martin-Grunewald commented:
@slava77 
Yes, you are correct: the error is the one with return code  21248, and they appear only in slc6_amd64_gcc493. By now it affects two IB builds, CMSSW_8_0_X_2016-04-18-1100
and CMSSW_8_0_X_2016-04-18-2300, so some intervention (cvmsf?) is still needed.
(The key 9 JEC error is another error).
